forge-gui/res/cardsfolder/upcoming/burakos_party_leader.txt

```diff
@@ -0,0 +1,13 @@
+Name:Burakos, Party Leader
+ManaCost:3 B
+Types:Creature Orc
+PT:2/4
+K:Choose a Background
+S:Mode$ Continuous | EffectZone$ All | Affected$ Card.Self | CharacteristicDefining$ True | AddType$ Cleric & Rogue & Warrior & Wizard | Description$ CARDNAME is also a Cleric, Rogue, Warrior, and Wizard.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigLoseLife | TriggerDescription$ Whenever CARDNAME attacks, defending player loses X life
+SVar:TrigLoseLife:DB$ LoseLife | Defined$ TriggeredDefendingPlayer | LifeAmount$ X | SubAbility$ DBTreasureTokens
+SVar:DBTreasureTokens:DB$ Token | TokenAmount$ X | TokenScript$ c_a_treasure_sac | TokenOwner$ You | Description$  and you create X Treasure tokens, where X is the number of creatures in your party.
+SVar:X:Count$Party
+DeckHints:Ability$Party & Type$Cleric|Rogue|Warrior|Wizard
+SVar:PlayMain1:TRUE
+Oracle:Burakos, Party Leader is also a Cleric, Rogue, Warrior, and Wizard.\nWhenever Burakos attacks, defending player loses X life and you create X Treasure tokens, where X is the number of creatures in your party.\nChoose a Background
```

Comment:
Might be worth adding:
DeckHas:Ability$Token|Sacrifice & Type$Cleric|Rogue|Warrior|Wizard|Treasure|Artifact